symengine/derivative.cpp

```diff
@@ -452,6 +452,38 @@ class DiffImplementation
                        arg->diff(x)));
     }
 
+    static RCP<const Basic> diff(const Erfinv &self, const RCP<const Symbol> &x)
+    {
+        RCP<const Basic> arg = self.get_args()[0];
+        return mul(
+            div(mul(sqrt(pi), exp(mul(erfinv(arg), erfinv(arg)))), integer(2)),
```

Comment:
store `erfinv(arg)` instead of calling `erfinv(arg)` twice.
Or call `pow(erfinv(arg), integer(2))`